cmd/backup/script.go

```diff
@@ -75,7 +70,7 @@ func newScript() (*script, error) {
 		stats: &Stats{
 			StartTime: time.Now(),
 			LogOutput: logBuffer,
-			Storages:  StoragesStats{},
+			Storages:  map[string]StorageStats{},
```

Comment:
Not sure if this has a real world impact, but technically, this is a breaking change. Assume the script fails as early as possible and these stats have never been written to, trying to access for example `.S3` in a templatr would create a nil dereference error when previously it would just return the zero values. If we wanted to play it safe, we could define empty values for all known storage backends here, even if it creates some duplication:

```go
map[string]StorageStats{
  "S3": StorageStats{},
  // ..and so on
}
```

What do you think? Can we somehow make this smarter and easier to maintain while still keeping the old behavior?

Reply:
I'm not sure how to automate this, but for now it's manually initialized, just to be safe.